task/buildah/0.1/buildah.yaml

```diff
@@ -45,6 +45,10 @@ spec:
     description: In case it is not empty, the prefetched content should be made available to the build.
     name: PREFETCH_INPUT
     type: string
+  - default: ""
+    description: Delete image tag after specified time. Empty means to keep the image tag.
```

Comment:
Does adding some examples for the values in the description will help like `time values could be something like 1h, 2d, 3w for hours, days, and weeks, respectively`?